osquery/tables/system/BUCK

```diff
@@ -280,6 +280,7 @@ osquery_cxx_library(
                 "windows/smbios_tables.cpp",
                 "windows/startup_items.cpp",
                 "windows/system_info.cpp",
+		"windows/userassist.cpp",
```

Comment:
nitpick, looks like this is missing a ` `.

Reply:
When i view the file I dont see a missing space?
The only time i see a missing space is in this comment?
Just confirm do u still see a missing space if u view the raw file?
If so i can add it, but the line looks correct when i view it